frontend/src/concepts/userSSAR/AccessReviewContext.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { checkAccess } from '~/api';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import useNamespaces from '~/pages/notebookController/useNamespaces';
+
+type AccessReviewCacheThingType = {
+  isLoading: boolean;
```

Comment:
should this be loaded instead to be the same as the hook and the checkAccess function

Reply:
I think everything should be `isLoading` and `hasAccess` -- price of using your code and the varying different ways other things have been written. I'll align them all on the `is` syntax as they are booleans.